pkg/scheduler/framework/plugins/selectorspread/selector_spread_test.go

```diff
@@ -396,7 +396,8 @@ func TestSelectorSpreadScore(t *testing.T) {
 		t.Run(test.name, func(t *testing.T) {
 			nodes := makeNodeList(test.nodes)
 			snapshot := cache.NewSnapshot(test.pods, nodes)
-			ctx := context.Background()
+			ctx, cancel := context.WithCancel(context.Background())
```

Comment:
Thanks @mengjiao-liu 
Can you help refine other part of this file?
https://github.com/kubernetes/kubernetes/blob/ba82add41a06d1fceeaf01fc71d3d5da910c41d9/pkg/scheduler/framework/plugins/selectorspread/selector_spread_test.go#L656

Reply:
Okay, I'll check the other tests for this file.

Reply:
Updated.

Reply:
Also help to refine other files like [`selector_spread_perf_test.go`](https://github.com/kubernetes/kubernetes/blob/master/pkg/scheduler/framework/plugins/selectorspread/selector_spread_perf_test.go#L61) as landed under the same package. thanks.

Reply:
Updated. Thanks!